plugins/kubernetes/app/models/kubernetes/cluster.rb

```diff
@@ -46,6 +46,14 @@ def kubeconfig
       @kubeconfig ||= Kubeclient::Config.read(config_filepath)
     end
 
+    def schedulable_nodes
+      client.get_nodes.
+        reject { |n| n.dig(:spec, :unschedulable) }.
+        select { |n| n.dig(:metadata, :labels, :"node-type") == "node" }
```

Comment:
We probably only need the first one, since the API hosts are always unscheduable, and `node-type: node` is something Zendesk-specific.